Index: axes.py
===================================================================
--- axes.py	(revision 3855)
+++ axes.py	(working copy)
@@ -1999,7 +1999,15 @@
 
 
     #### Labelling
+    
+    def get_title(self):
+        """
+        GET_TITLE():
 
+        Get the title for the axes.  
+        """
+        return self.title.get_text()
+
     def set_title(self, label, fontdict=None, **kwargs):
         """
         SET_TITLE(label, fontdict=None, **kwargs):
@@ -2024,7 +2032,16 @@
         self.title.update(kwargs)
         return self.title
     set_title.__doc__ = cbook.dedent(set_title.__doc__) % martist.kwdocd
+    
+    def get_xlabel(self):
+        """
+        GET_XLABEL()
 
+        Get the label for the xaxis.
+        """
+        label=self.xaxis.get_label()
+        return label.get_text()
+    
     def set_xlabel(self, xlabel, fontdict=None, **kwargs):
         """
         SET_XLABEL(xlabel, fontdict=None, **kwargs)
@@ -2043,7 +2060,16 @@
         label.update(kwargs)
         return label
     set_xlabel.__doc__ = cbook.dedent(set_xlabel.__doc__) % martist.kwdocd
+    
+    def get_ylabel(self):
+        """
+        GET_YLABEL()
 
+        Get the label for the yaxis.
+        """
+        label=self.yaxis.get_label()
+        return label.get_text()
+    
     def set_ylabel(self, ylabel, fontdict=None, **kwargs):
         """
         SET_YLABEL(ylabel, fontdict=None, **kwargs)
